andela/ah-jawans-frontend
# #166790061 user should be able to view notifications.

#### What does this PR do?

Allow user to view notifications in real time

#### Description of Task to be completed?

Users should be able to view notifications

#### How should this be manually tested?

Login to Authors haven the check your profile picture in the top right side of the home page and a bell icon after that.

After cloning the repo, cd into it, type npm run test. To run the application type npm start

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166790061

#### Screenshots (if appropriate)
